### PR TITLE
Use babel-preset-env to manage what gets transpiled and use postinstall to build after npm install

### PR DESCRIPTION
**Pull request by brendanvinson**
Ref: https://github.com/jeffijoe/koa-es7-boilerplate/issues/6

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "test-watch": "npm run test -- --watch",
     "lint": "eslint src/ test/",
     "lint-watch": "esw -w src/ test/",
-    "cover": "cross-env NODE_ENV=test nyc npm test"
+    "cover": "cross-env NODE_ENV=test nyc npm test",
+    "postinstall": "npm run build"
```
Comment (jeffijoe) [inline package.json]: Why this though?

Comment (brendanvinson) [reply package.json]: It's an efficiency thing so you don't have to run `npm build` separately after `npm install`; instead it'll run it automatically. You can always run it using just the `npm build` command as well if you choose.

Not a big deal, I can take it out if you don't like it in there.

Comment (jeffijoe) [reply package.json]: Right, but when you pull down the project and install deps to start working on it, you would run `npm run dev`. I don't think we should put a `postinstall` there. 😄

Comment (brendanvinson) [reply package.json]: Ah right because of babel-node. My bad.

Comment (jeffijoe) [reply package.json]: Winner winner chicken dinner! 😄

**Pull request merged**